bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/service/LegalEntityService.kt

```diff
@@ -58,7 +63,7 @@ class LegalEntityService(
         legalEntities.forEach { legalEntity ->
             changelogRepository.save(ChangelogEntry(legalEntity.externalId, LsaType.LegalEntity))
         }
-        legalEntityPersistenceService.persistLegalEntitiesBP(legalEntities);
+        legalEntityPersistenceService.persistLegalEntitiesBP(legalEntities)
     }
 
     fun getLegalEntityByExternalId(externalId: String): LegalEntityGateInputResponse {
```

Comment:
This method fetches from SaaS still. Should also use the database to fetch the business partner